morse.rb

```diff
@@ -0,0 +1,46 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(char)
+  MORSE_CODE[char]
+end
+
+def decode(line)
+  words = line.split('   ')
+  decoded_words = words.map do |word|
+    word.split.map { |char| decode_char(char) }.join
+  end
+  decoded_words.join(' ')
+end
```

Comment:
- [ ] You did a great job creating the method to decode Morse code. :+1: There seems to be one function missing, which is supposed to decode a word. Please implement such a function. You can move some of the lines from the `decode` function to achieve it. 😎️

**Requirements**
![image](https://github.com/Odongo006/Morse-Code/assets/17243325/21508d88-75a0-4e1e-b5ec-a62bbaeeb21f)

Reply:
Hi @amejid In our code if you want to decode word("-- -.--") it will return "MY" , i don't understand what you ask for?

Reply:
Hey @Hisoka37, apparently the `decode` function in your project does both things, it decodes a word and sentence. Please refactor and break it up into two functions, one to decode a word and one to decode an entire message. The requirement suggests that we should have three separate functions. Happy coding. 🤝️

**Project requirements**
![image](https://github.com/Odongo006/Morse-Code/assets/17243325/7b9b612a-bb49-44de-8b29-ef687ce332e3)